connect-file-pulse-plugin/pom.xml

```diff
@@ -381,6 +381,10 @@
             <groupId>io.rest-assured</groupId>
             <artifactId>json-path</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
```

Comment:
Can we check if the upstream project is also using assertj? Otherwise we are introducing some dependency we could avoid by refactoring tests a bit

Reply:
they are not useing assertj at the moment, but thougt this may be ok for test dependencies only, may worth asking their feedback